__tests__/app.tests.js

```diff
@@ -0,0 +1,37 @@
+const app = require('../app');
+const request = require('supertest');
+const data = require('../db/data/test-data');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection.js');
+
+afterAll(()=>{
+    return db.end();
+})
+beforeEach(()=>{
+    return seed(data)
+})
+
+describe('app',()=>{
+    test('responds with status of 200 and object with description and slug keys',()=>{
+        return request(app).get('/api/topics').expect(200)
+        .then((response)=>{
+            const {topics} = response.body
+            topics.forEach((topic)=>{
+                expect(topic).toHaveProperty('description',expect.any(String));
+                expect(topic).toHaveProperty('slug',expect.any(String));
+            })
+        })
```

Comment:
This is good Sula, checking the structure of the data coming back from `/api/topics` is the right approach.